account_global_discount/hooks.py

```diff
@@ -0,0 +1,23 @@
+from odoo.tools.sql import column_exists
+
+
+def _pre_init_global_discount_fields(cr):
+    if not column_exists(cr, "account_move", "amount_global_discount"):
+        cr.execute(
+            """
+                ALTER TABLE "account_move"
+                ADD COLUMN "amount_global_discount" double precision
```

Comment:
No 0 initialization?

Reply:
Sorry, I added default and drop after, thank you